### ISSUE TITLE
V 8.8.10 nefunguje

### ISSUE THREAD

**Issue by kallenp** [Screemy/zimbra-czech-translation#1]
V 8.8.10 nefunguje

**Comment by ZS-Man** [comment]
Zdravim, koukam ze ty soubory s prekladem jsou prazdne.....

**Comment by kallenp** [comment]
A má to řešení  ?

**Comment by ZS-Man** [comment]
No, asi cekat jestli si toho autor vsimne a soubory sem nahraje.

**Comment by Screemy** [comment]
ano, vim o tom, projekt musel jit ale bohuzel stranou, kvuli jinym dulezitejsim projektum. Kazdopadne to mam v planu dodelat

**Comment by Screemy** [comment]
Sice po dělší době, ale hotovo na aktualní verzi.
